src/main/java/jarvis/Jarvis.java

```diff
@@ -0,0 +1,95 @@
+package jarvis;
+
+import command.Parser;
+
+import services.TextUi;
+import services.bizerrors.JarvisException;
+import services.tasklist.Storage;
+import services.tasklist.TaskList;
+
+import java.util.Scanner;
+
+public class Jarvis {
+    public final static String JARVIS_LOGO = "                                                                 \n" +
```

Comment:
Nice that you remembered to make constant names uppercase with underscores

Reply:
Thanks!